airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/spec.json

```diff
@@ -14,6 +14,26 @@
         "description": "Choose how to authenticate to Lever Hiring.",
         "type": "object",
         "oneOf": [
+          {
+            "type": "object",
+            "title": "Authenticate via Lever (Api Key)",
+            "required": ["api_key"],
+              "properties": {
+                "auth_type": {
+                  "type": "string",
+                  "const": "Api Key",
+                  "enum": ["Api Key"],
+                  "default": "Api Key",
+                  "order": 0
+                },
+                "api_key": {
+                  "title": "Api key",
+                  "type": "string",
+                  "description": "The Api Key of your Lever Hiring account.",
+                  "airbyte_secret": true,
+                  "order":1}
```

Comment:
can you add a new line for the bracket parenthesis `}`?

```
  "airbyte_secret": true,
  "order": 1
}
```

Reply:
done